src/components/settings-row/settings-row.js

```diff
@@ -60,9 +60,12 @@ class SettingsRow extends React.Component {
      * Heading description
      *
      * @property  description
-     * @type      {Node}
+     * @type      {String|Object}
      */
-    description: PropTypes.node,
+    description: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.object
```

Comment:
Just curious about this change? Isn't `node` more flexible?

Reply:
I suppose so. And I suppose there is a risk of this introducing a breaking change.

However, would the `description` be set to anything other than a string or an JSX object?
And we use `PropTypes.oneOfType([PropTypes.string, PropTypes.object])` elsewhere e.g. in `Heading`: https://github.com/Sage/carbon/blob/master/src/components/heading/heading.js#L36